public/js/analytics/colorCount.js

```diff
@@ -0,0 +1,113 @@
+function areArraysEqualSets(a1, a2) {
```

Comment:
This function can be implemented more concisely as follows:
```javascript
const set1 = new Set(a1);
const set2 = new Set(a2);
return a1.every((x) => set2.has(x)) && a2.every((x) => set1.has(x));
```
If this code is all in webpack land, that function could be in a utility file as well.